Project/QtCreator/QCTools.pro

```diff
@@ -99,10 +120,15 @@ LIBS      += -L$${PWD}/../../../ffmpeg/libavdevice -lavdevice \
              -L$${PWD}/../../../ffmpeg/libswscale -lswscale \
              -L$${PWD}/../../../ffmpeg/libavcodec -lavcodec \
              -L$${PWD}/../../../ffmpeg/libavutil -lavutil
-LIBS      += -L$${PWD}/../../../freetype/usr/lib -lfreetype
-LIBS      += -lbz2
 
-!macx:LIBS      += -ldl -lrt
+!win32 {
+    LIBS      += -L$${PWD}/../../../freetype/usr/lib -lfreetype
+    LIBS      += -lbz2
+}
+
+unix {

```

Comment:
This line should be `linux` instead of `unix` because mac os x doesn't have librt (and doesn't need libdl).